task/01-strings-tasks.js

```diff
@@ -238,7 +243,7 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    return typeof value === 'string' | value instanceof String;
 }
 
```

Comment:
It works fine, but looks like it is not what you mean )
https://learn.javascript.ru/bitwise-operators